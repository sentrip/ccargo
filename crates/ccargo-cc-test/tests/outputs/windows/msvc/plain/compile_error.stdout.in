main.c
main.c(4,9): error C2120: 'void' illegal with all types
main.c(5,9): error C2120: 'void' illegal with all types
