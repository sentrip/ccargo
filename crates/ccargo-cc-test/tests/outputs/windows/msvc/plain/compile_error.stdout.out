error C2120: 'void' illegal with all types
   --> main.c:4:9
error C2120: 'void' illegal with all types
   --> main.c:5:9
